### DISCUSSION TITLE
Why client doesnt send farmer space info to pools?

### DISCUSSION THREAD

**Discussion by MaximusChia** [Chia-Network/chia-blockchain#8201] [Ideas]
Everytime we see values fluctiating over time, in pools. Thus making space info not so precise.
Wonder why client dont take farmer space data and send to the pools? What prevents doing it?
Estimates are currently done by partials and points sent. Which are not exact everytime.

**Comment by emlowe** [comment]
Because it's trivial for a client to simply state it has 300Tib when it only has 5Tib - and therefore it's not trustworthy information
However, the number of partials cannot be faked and is the information the pool uses to calculate your payout

(The client could send the # of plots as a 'hint' for the farm, but it would have no impact on your actual payout - and wouldn't mean anything, as such most pools would likely just choose to ignore it)

**Reply by MaximusChia** [reply]
I got it, partials is what matters, but then both data can be used to determine the validity of farmers plots. Instead of just determining the space from how many partials are sent.
You have X number of plots or TiB, and must correlate with how many partials you sending. Could this prevent any fake space?
I consider this an improvement to the system.

**Comment by BrandtH22** [comment]
@MaximusChia , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
